CHANGELOG.md

```diff
@@ -10,7 +10,11 @@ We use semantic versioning in some slight variation until our feature set has st
 
 After this we will switch probably to real [Semantic Versioning 2.0.0](http://semver.org/)
 
-### 4.0.0-M3
+### 4.0-SNAPSHOT
```

Comment:
Can you change it to 4.1-SNAPSHOT ?

Reply:
pom is still not updated to 4.1-SNAPSHOT

Reply:
anyway we are going to change it sooner or later. let me update